CMakeLists.txt

```diff
@@ -116,7 +126,7 @@ set(obs-browser_HEADERS
 
 # only allow browser panels on win32 for now -- other operating systems
 # need more testing
-if(WIN32 AND BROWSER_PANEL_SUPPORT_ENABLED)
+if(BROWSER_PANEL_SUPPORT_ENABLED)
```

Comment:
This seems unrelated to the PR. Did you mean to add this change?

Reply:
woops, that was unintended. I corrected it